src/storage/src/hummock/iterator/forward_user.rs

```diff
@@ -321,7 +312,29 @@ impl UserIterator {
     }
 }
 
-impl DirectedUserIteratorBuilder for UserIterator {
+#[cfg(test)]
+impl UserIterator<UnorderedMergeIteratorInner<UserIteratorPayloadType<Forward, SstableIterator>>> {
```

Comment:
we can use a type define to simplify `UnorderedMergeIteratorInner<UserIteratorPayloadType<Forward, SstableIterator>>`